in_toto/matchrule_validators.py

```diff
@@ -25,7 +25,8 @@ def _validate_match_rule(keywords):
 
   if not isinstance(keywords, list):
     raise securesystemslib.exceptions.FormatError(
-        "this matching rule is not a list")
+        "Wrong rule format, must be a list.\n\t"
```

Comment:
Re header block: Thanks, will fix.
Re indentation: I operate according to our code style guide lines here, which suggests [double-indentation for line continuation](https://github.com/secure-systems-lab/code-style-guidelines#maximum-line-length)